exporter/trace/stdout/stdout_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"testing"
 	"time"
 
+	"go.opentelemetry.io/api/trace"
```

Comment:
This should be grouped together with the import of "go.opentelemetry.io/{api/core,sdk/export}" below.

Reply:
fixed :)